src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,46 @@
-import React from 'react';
-
+import { Movie } from '../../types/Movie';
```

Comment:
on types exit just `movie` file, not `Movie`

Reply:
@AlinaMatuschak It was strange, cause I have correct filename locally. It seems that it is a Github issue. Fixed by toggling filename with git pushes.

Reply:
maybe you didn’t push Movie file